runner/runner.go

```diff
@@ -1329,6 +1330,48 @@ func Grade(
 						},
 					)
 				}
+				// Check if we have an expected stderr message from the validator;
+				// fail the case with JE if there's a mismatch.
+				if runScore.Cmp(big.NewRat(1, 1)) != 0 {
```

Comment:
```suggestion
				if runScore.Cmp(big.NewRat(0, 1)) == 0 {
```

aquí no deberíamos compararlo con cero?

Reply:
la semántica que tenía en mente es que cualquier no-AC debería explicar por qué no dio lo que debía (y entonces hay que checar stderr siempre que el score no sea 1)

Reply:
suena razonable! puedes actualizar el comentario entonces para que esto sea explícito?